x-pack/plugins/endpoint/public/applications/endpoint/store/alerts/reducer.ts

```diff
@@ -60,6 +68,20 @@ export const alertListReducer: Reducer<AlertListState, AppAction> = (
         patterns: action.payload,
       },
     };
+  } else if (action.type === 'serverSuccessfullyClosedAlert') {
+    return {
+      ...state,
+      closedAlert: {
+        success: true,
```

Comment:
I think you may need to `...closedAlert` to get the error property in there too.

Reply:
that spread doesn't work but i put something in there that has the same effect